app/models/journeys/early_years_payment/provider/authenticated/slug_sequence.rb

```diff
@@ -26,6 +26,11 @@ class SlugSequence
 
           RESTRICTED_SLUGS = [].freeze
 
+          DEAD_END_SLUGS = %w[
```

Comment:
I'm hoping at some point in the future we can remove the `DEAD_END_SLUGS` constants now that we have a state machine-esque setup and once the positive slugs sequences are in place

Reply:
We still need DEAD_END_SLUGS to determine when to render a back link or not, though there's probably a better approach for that